src/lib.rs

```diff
@@ -475,6 +507,15 @@ macro_rules! impl_integer_for_isize {
                 assert_eq!((11 as $T).lcm(&5), 55 as $T);
             }
 
+            #[test]
+            fn test_gcd_lcm() {
+                for i in 1..=255 {
+                    for j in 0..=255 {
```

Comment:
I think it will be fine to use a plain range for compatibility.

Maybe I'm missing something, but I'm surprised that this didn't warn about 255 being out of range for `i8`.  But anyway, if you want to be exhaustive, perhaps we should use negative values too with `-128..127`.

I also notice that you avoid `lcm(0, 0)`, but I think this is actually a bug in our existing implementation.  We would get a divide-by-zero since `gcd(0, 0) = 0`, but Wolfram Alpha says [`lcm(0, 0) = 0`](http://www.wolframalpha.com/input/?i=lcm(0,0)) too.